lambdas/store-identity/src/main/java/uk/gov/di/ipv/core/storeidentity/StoreIdentityHandler.java

```diff
@@ -117,7 +116,15 @@ public Map<String, Object> handleRequest(ProcessRequest input, Context context)
                     evcsService.storePendingIdentity(
                             userId, credentials, clientOAuthSessionItem.getEvcsAccessToken());
                 }
+                credentials.forEach(
+                        credential -> {
+                            if (credential.getMigrated() == null)
+                                credential.setMigrated(Instant.now());
```

Comment:
I think we actually want to do this even if there is already a migrated timestamp - it'll be more useful to have the date as the most recent time we know it was written to the store (rather than just the first time).

Reply:
I thought we want to keep when this was first migrated.

Reply:
I think the situation where a timestamp is useful is something like:
1. We turn on EVCS migrations and records start getting written into EVCS
2. Something goes wrong, and we end up turning it off and wiping EVCS
3. We turn it back on again and start writing records again

In this case all records with a timestamp older than step 2 are actually not migrated, but if we write them again then we want to update the timestamp to indicate that they have been again.

Does that make sense? It's all a bit hypothetical - if all goes well we won't need this!

Reply:
done.